win32ss/gdi/gdi32_vista/d3dkmt.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * PROJECT:     ReactOS Display Driver Model
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     D3DKMT dxgkrnl syscalls
+ * COPYRIGHT:   Copyright 2023 Justin Miller <justin.miller@reactos.org>
```

Comment:
Is this code that old, or did you forget to update this after copying the header?

Reply:
This PR was opened in 2023 :)